packages/xo-server/src/xo-mixins/backups-ng-logs.js

```diff
@@ -1,12 +1,18 @@
 import ms from 'ms'
 import { forEach, isEmpty, iteratee, sortedIndexBy } from 'lodash'
+import { noSuchObject } from 'xo-common/api-errors'
 
 import { debounceWithKey } from '../_pDebounceWithKey'
 
 const isSkippedError = error =>
   error.message === 'no disks found' ||
   error.message === 'no VMs match this pattern' ||
-  error.message === 'unhealthy VDI chain'
+  error.message === 'unhealthy VDI chain' ||
+  noSuchObject.is(error, {
+    data: {
+      type: 'VM',
+    },
+  })
```

Comment:
Do we want the state to be skipped or failure?

What about `backup-reports`?

Reply:
You're right, after though it should be `failure` because it's critical for users.